app/src/main/java/de/westnordost/streetcomplete/quests/cycleway/AddCycleway.kt

```diff
@@ -164,7 +164,7 @@ private val untaggedRoadsFilter by lazy { """
 """.toElementFilterExpression() }
 
 private val notInZone30OrLess = """
-   ~"${(MAXSPEED_TYPE_KEYS + "maxspeed").joinToString("|")}" ~ ".*:(urban|rural|trunk|motorway|nsl_single|nsl_dual)"
+   !(~"${(MAXSPEED_TYPE_KEYS + "maxspeed").joinToString("|")}" ~ ".*:(30|20)")
```

Comment:
The filter should be like in `AddRoadWidth` or `CyclewayOverlay`:

`".*:(zone)?:?([1-9]|[1-2][0-9]|30)"`